bokeh/models/widgets/tables.py

```diff
@@ -239,6 +241,28 @@ class BooleanFormatter(CellFormatter):
     The icon visualizing the check mark.
     """)
 
+class CompositeFormatter(CellFormatter):
+    ''' CompositeFormatter applies multiple formatters to a single TableColumn
+
+    Example:
+
+    .. code-block:: python
+
+        example_formatter = CompositeFormatter(formatters=[StringFormatter(font_style="bold"), HTMLTemplateFormatter(
+            template='<code><%= value %></code>')])
+
+        TableColumn(field='Part ID', title='Part ID', formatter=example_formatter, ...)
+    '''
+
+    formatters = Seq(Instance(CellFormatter, help="""
+        The CompositeFormatter class may be used with any combination of
+        the installed formatters in the Bokeh library to apply multiple
+        formatting capabilities (BooleanFormatter, CellFormatter, DateFormatter,
+        HTMLTemplateFormatter, NumberFormatter, ScientificFormatter, StringFormatter,
+        to a single TableColumn. Formatters will be implemented in the same order they appear
+        in the sequence.
```

Comment:
editing suggestion:
```
   formatters = Seq(Instance(CellFormatter, help="""
        A list of CellFormatter instances (e.g. DateFormatter, HTMLTemplateFormatter, etc)
        to apply in succession to a TableColumn. Formatters are applied in the order that
        they are given here. 
    """)
```

Reply:
The first part "The CompositeFormatter class may be used..." would be more suitable in the top-level docstring, IMO